### PR TITLE
Add healthcheck method to the Tree service

### LINKED ISSUE #1768: Tree service discovery
Tree service is an optional service. Not sure it will stay optional, but now it is. Clients want to discover if tree service is enabled on remote node for early failures (see https://github.com/nspcc-dev/neofs-s3-gw/issues/699). Clients can't rely on `Dial` invocations, because tree service lives on the same socket as NeoFS API. 

I see such options how tree service can be discovered
1) Provide dummy RPC to check if server returns error
2) Use existing RPC as dummy to check if server returns error
3) Enable [server reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md).

I think (3) is an overhead and more suitable for debug purposes rather than using it in production.
(1) looks better than (2) for me, but requires protocol update.

### PR DESCRIPTION
**Pull request by acid-ant**
Closes #1768 

Signed-off-by: Anton Nikiforov <an.nikiforov@yadro.com>

### CODE REVIEW

File: pkg/services/tree/service.proto [hunk 1]
```diff
@@ -47,6 +48,8 @@ service TreeService {
   rpc Apply (ApplyRequest) returns (ApplyResponse);
   // GetOpLog returns a stream of logged operations starting from some height.
   rpc GetOpLog(GetOpLogRequest) returns (stream GetOpLogResponse);
+  // Healthcheck is a dummy rpc to check is service available
+  rpc Healthcheck(google.protobuf.Empty) returns (google.protobuf.Empty);
```
Comment (carpawell) [inline pkg/services/tree/service.proto:52]: we usually [add](https://github.com/nspcc-dev/neofs-node/blob/2d43892fc9bfc0067689b5ef1370873c9bb5a7b1/pkg/services/tree/service.proto#L140) empty request structs explicitly for clear extensibility

/cc @fyrchik, @cthulhu-rider

File: pkg/services/tree/service.proto [hunk 2]
```diff
@@ -47,6 +48,8 @@ service TreeService {
   rpc Apply (ApplyRequest) returns (ApplyResponse);
   // GetOpLog returns a stream of logged operations starting from some height.
   rpc GetOpLog(GetOpLogRequest) returns (stream GetOpLogResponse);
+  // Healthcheck is a dummy rpc to check is service available
```
Comment (carpawell) [inline pkg/services/tree/service.proto:51]: please, double check that the comment is correct, seems like it should be `check if ... is ...` construct

Comment (acid-ant) [reply pkg/services/tree/service.proto]: Thanks, rephrased.

File: pkg/services/tree/service.proto (hunk 1 above)
Comment (acid-ant) [reply pkg/services/tree/service.proto]: Added custom response/request.

**Review by cthulhu-rider — DISMISSED** [review]
Pls pay attention to missing linebreak in `.proto` file.

**Review by carpawell — APPROVED** [review]
Except for the missing empty line.

**Review by carpawell — APPROVED** [review]

**Review by fyrchik — APPROVED** [review]

**Review by alexvanin — APPROVED** [review]

**Pull request merged**